source/accnt_mgmt.rst

```diff
@@ -164,11 +160,15 @@ Would yield the following response:
 
 .. only:: nodejs
 
+  Would yield the following response:
```

Comment:
Since the nodejs and php sections were already relying on the fact that "Would yield the following response" was above, I copied it here.